telegraf/Dockerfile

```diff
@@ -33,11 +33,16 @@ RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y --no-ins
     apt-get clean && \
     rm -rf /var/lib/apt/lists/*
 
-RUN git clone https://gitlab.freedesktop.org/drm/igt-gpu-tools.git; cd igt-gpu-tools; git checkout 2b29e8ac07fbcfadc48b9d60e4d736a6e3b289ab
-RUN cd igt-gpu-tools; meson build; ninja -C build; cd build; ninja install    
+WORKDIR /igt
+
+# Install 1.27.1
+RUN git config --global http.proxy $HTTP_PROXY; git clone https://gitlab.freedesktop.org/drm/igt-gpu-tools.git; cd igt-gpu-tools; git checkout 	2b29e8ac07fbcfadc48b9d60e4d736a6e3b289ab
+
+## JSON output: /usr/local/bin/intel_gpu_top -d pci:card=$1 -J 
+RUN cd  igt-gpu-tools; meson build; ninja -C build; cd build; ninja install
 
 # Build and install telegraf/influxdb: Reference: https://docs.influxdata.com/telegraf/v1.21/introduction/installation/
-WORKDIR /opt/intel-gpu-telegraf
+WORKDIR ./opt/intel-gpu-telegraf
```

Comment:
Would the full path for this be /igt/opt/intel-gpu-telegraf?

Reply:
I just removed it. It was unnecessary.